README.md

```diff
@@ -5,10 +5,14 @@
 
 ## A Specification Framework for the JVM
 
+Visit the [web site](https://jetbrains.github.io/spek) for more documentation and information
 
+## Current State of the Project
 
-Visit the [web site](https://jetbrains.github.io/spek) for more documentation and information
+- Spek 1.x is in support mode, only bug fixes and documentation updates will be merged into 1.x branch.
+- Spek 2.x is main development branch.
```

Comment:
Probably it would be helpful to state `2.x` goals or link to the milestone.

Reply:
Added link to milestone, I didn't do it because it was something more suitable for 2.x branch or website, but ok